FlowCrypt/Controllers/Settings/KeySettings/Key List/Helpers/KeySettingsProvider.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  KeySettingsProvider.swift
+//  FlowCrypt
+//
+//  Created by Anton Kharchevskyi on 12/13/19.
+//  Copyright © 2019 FlowCrypt Limited. All rights reserved.
+//
+
+import Foundation
+
+protocol KeySettingsProviderType {
```

Comment:
Is protocol useful if we only plan one implementation?

Reply:
Sure, for unit tests.

Reply:
When we are going to start writing tests we can create some mock providers. Having a protocol is basic for Depedecey injection and possibility to write tests